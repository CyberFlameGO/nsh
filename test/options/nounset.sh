# exit-with=1
defined=123

echo $defined
echo $undefined

echo "enabling nounset"
set -u

echo $defined
echo $undefined

echo "unreachable"

