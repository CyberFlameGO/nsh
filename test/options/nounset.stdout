123

enabling nounset
123
